controller.go

```diff
@@ -394,6 +394,12 @@ func (c *controller) ReloadConfiguration(cfgOptions ...config.Option) error {
 	c.cfg = cfg
 	c.Unlock()
 
+	if c.discovery == nil && c.cfg.Cluster.Watcher != nil {
```

Comment:
not part of our changes (see https://github.com/medallia/libnetwork/commit/9876affaed02879b20a954b9fc9c75b32e513308)